easybuild/tools/options.py

```diff
@@ -562,6 +566,10 @@ def github_options(self):
             'github-user': ("GitHub username", str, 'store', None),
             'github-org': ("GitHub organization", str, 'store', None),
             'install-github-token': ("Install GitHub token (requires --github-user)", None, 'store_true', False),
+            'list-prs': ("List pull requests", None, 'store_true', None),
+            'list-prs-state': ("State for listing pull requests", 'choice', 'store', 'open', github_list_pr_states),
+            'list-prs-order': ("Order for sorting pull requests", 'choice', 'store', 'created', github_list_pr_orders),
+            'list-prs-direction': ("Direction for sorting pull requests", 'choice', 'store', 'desc', github_list_pr_directions),
```

Comment:
@migueldiascosta Do you think it makes sense to make this a single option, that accepts three values (optionally) for state/order/direction?

Something like:

```
eb --list-prs  # list all open PRs, order by created (desc.)
eb --list-prs closed  # list all closed PRs, default order
eb --list-prs all:updated  # list all PRs, order by updated (default ordering)
eb --list-prs open:updated:asc